### ISSUE TITLE
Update VZA shifts 07/17/23-10/08/23

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#12781]
## Description
Add the next 12-weeks of assignments. 

## Note
Current need – the last copy of shifts seems to have not worked well.  Current shifts are only two hours long compared to the 6-8 hour shifts we had back earlier this year. Can you recopy for the full month of July and August from an earlier 8-week stretch that had the longer shifts?

**Comment by patrickm02L** [comment]
Identified the period between 1/30-4/23 to be the 12-week batch of assignments to repeat.

After 4/23 is when the shifts got shrunken for some reason.

**Comment by ChrispinP** [comment]
I found the bug with the previous dataset, some of the years were set as 2020 instead of 2023 in Excel. A bit late but I have updated the existing records so they are correct now with 2023 and will show their correct duration on the calendar. Ive also updated the documentation to help prevent this error in the future by separating the Assignment Date Time field into 6 separate columns when building the dataset in Excel now instead of just 4.

The next 12 weeks (285 records) have been added for 07/17/23 to 10/08/23

Complete
